src/components/OIO/defaults.js

```diff
@@ -0,0 +1,68 @@
+export default {
+   base: {
+      colors: {
+         primary: '#78909C',
+         none: 'none'
+      },
+      elementHeight: {
+         large: '48px',
+         medium: '36px',
+         small: '30px',
+         tiny: '24px'
+      }
+   },
+   font: {
+      family: 'Helvetica Neue, Arial, sans-serif',
+      sizes: {
+         1: {
+            fontSize: '9px[a] 10px[b-e]',
+            lineHeight: '16px'
+         },
+         2: {
+            fontSize: '13px',
+            lineHeight: '17px'
+         },
+         3: {
+            fontSize: '15px',
+            lineHeight: '21px'
+         },
+         4: {
+            fontSize: '17px',
+            lineHeight: '22px'
+         },
+         5: {
+            fontSize: '19px',
+            lineHeight: '23px'
+         },
+         6: {
+            fontSize: '25px',
+            lineHeight: '30px'
+         },
+         7: {
+            fontSize: '34px',
+            lineHeight: '1.2'
+         },
+         8: {
+            fontSize: '45px',
+            lineHeight: '1.2em'
+         },
+         9: {
+            fontSize: '56px',
+            lineHeight: '1.1em'
+         },
+         10: {
+            fontSize: '78px',
+            lineHeight: '78px'
+         }
+      }
+   },
+   viewport: {
+      breakpoints: {
+         a: '0px',
+         b: '768px',
+         c: '992px',
+         d: '1300px',
+         e: '1650px'
```

Comment:
I'm guessing this is using characters instead of numbers because of our breakpoint system that uses the `2[b-e]` style syntax?